package.json

```diff
@@ -48,6 +49,9 @@
         "start-tests": "webpack-dev-server -d --config webpack.tests.config.js",
         "test-browser": "ts-node -- ./scripts/with-http-server.ts -C dist/test -p 8079 -- mocha-webdriver-runner http://localhost:8079/index.html",
         "build-tests": "webpack -d --config webpack.tests.config.js",
+        "run-rendering-tests": "./scripts/rendering-tests-reference-platform.sh",
+        "save-reference": "npx ts-node @here/harp-test-utils/lib/rendering/RenderingTestResultCli.ts save-reference",
```

Comment:
I think the naming is not consistent as just by looking at the name you won't notice they are in fact used in combination.
What about?
* run-rendering-tests
* save-reference-rendering-tests
* approve-reference-rendering-tests